atomic_defi_design/assets/languages/atomic_defi_ru.ts

```diff
@@ -171,22 +171,22 @@
     <message>
         <location filename="../../qml/Wallet/AddressBook.qml" line="47"/>
         <source>Search a contact by name or tags</source>
-        <translation>Поиск контакта по имени или тегам </translation>
+        <translation>Поиск контакта по имени или тегам</translation>
     </message>
     <message>
         <location filename="../../qml/Wallet/AddressBook.qml" line="81"/>
         <source>Name</source>
-        <translation type="unfinished">Название</translation>
+        <translation>Имя</translation>
     </message>
     <message>
         <location filename="../../qml/Wallet/AddressBook.qml" line="90"/>
         <source>Tags (first 6)</source>
-        <translation type="unfinished"></translation>
+        <translation>Тэги (первые 6)</translation>
```

Comment:
above `теги` spelling used
https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/863/files#diff-bd6de22ea5ed42741c56c99ae64a1f6e314593b3a5dd17e6f105ae814a1f8840R174

Reply:
sry, but I do not understand what you mean

Reply:
I meant that above ` <translation>Поиск контакта по имени или тегам</translation>` using `тЕги` spelling, and here `тЭги` - since it's english word I believe both spelling correct but it's better to use same in both places